src/main/java/globalquake/ui/globe/GlobePanel.java

```diff
@@ -121,15 +121,8 @@ public void mouseClicked(MouseEvent e) {
         });
 
         addMouseWheelListener(e -> {
-            boolean down = e.getWheelRotation() < 0;
-            double mul = down ? (1 / 1.15) : 1.15;
-
-            if (down) {
-                if (scroll >= 0.01)
-                    scroll *= mul;
-            } else if (scroll < 4) {
-                scroll *= mul;
-            }
+            double delta = 0.05f * e.getPreciseWheelRotation();
```

Comment:
@xspanger3770 thanks for the info, do you have any suggestions/recommendations on how you would like for this to be implemented? I can give it another try.

Reply:
Hi!
Regarding this small snippet of code, I don't really think there is much to improve than perhaps naming the variables a little better and replacing getWheelRotation with your suggestion .getPreciseWheelRotation for users with smooth scroll wheel.

But what I actually wanted to implement (and I should've probably explained it a bit more :D) is a more smooth zooming on the globe map even for users with scroll wheels with notches. The idea is that as soon as the mouse wheel event is triggered, instead of the scroll variable jumping directly to the next level, it would be progressively increased to the next value and thus creating a smooth zooming effect instead of flick.

I suggest running globalquake/ui/debug/GlobePanelDebug.java and playing around with any approach you come with.

Thank you for your keen interest in my project! I appreciate your enthusiasm and effort in contributing. If you have any further questions or need clarification, please don't hesitate to reach out. I'm here to help and support you in any way I can.

Reply:
Ah understood. I do see a big difference after replacing getWheelRotation with getPreciseWheelRotation. Will try to fine tune the incrementing value (mul) and see if that helps. Thanks for the info!

Reply:
I merged the current changes so that they can be included in the next release.
If you wish to add further enhancements to the scrolling, please feel free to initiate a new pull request. 
Wishing you the best of luck in your endeavors.

Reply:
@xspanger3770 sounds good. thanks for merging the pr.

Reply:
You can also try something like this:
```
double rotation = e.getPreciseWheelRotation();
boolean down = rotation < 0;

double delta = 1 + Math.abs(rotation) * 0.15;
double mul = down ? 1 / delta : delta;
...
```

Reply:
sure will give it a try, thanks!